src/test/java/entities/ExporterTest.java

```diff
@@ -54,45 +63,106 @@ public ExporterTest() {
         items.add(item9);
         items.add(item10);
 
-        inventory.setItems(items);
-    }
 
-    @Test
-    public void AssertUsrInvCorrect() throws IOException {
-        Exporter exporter = new Exporter(inventory);
-        exporter.export();
-        try {
-            FileReader fr_usr = new FileReader("src/main/java/exports/user_inventory.txt");
-            int i;
-            while ((i = fr_usr.read()) != -1) {
-                System.out.print((char) i);
-            }
-            System.out.println();
-            fr_usr.close();
-        }
-        catch (Exception e) {
-            e.getStackTrace();
+        inventoryTable = new String[items.size()][7];
+        for (int i = 0; i < items.size(); i++) {
+            InventoryItem item = (InventoryItem) items.get(i);
+            inventoryTable[i] = new String[]{item.getName(), item.getQuantity() + "", item.getBarcode(),
+                    item.getBuyPrice() + "", item.getSellPrice() + "",
+                    Integer.toString(item.getCaseQuantity()), item.getDepartment()};
         }
-        Assertions.assertTrue(true);
+        this.inventoryViewModel = new InventoryViewModel(inventoryTable, new String[]{});
+
+        ArrayList<InventoryItem> items2 = new ArrayList<>();
+        ArrayList<Order> orders = new ArrayList<>();
+        items2.add(item0);
+        items2.add(item1);
+        items2.add(item2);
+        items2.add(item3);
+        items2.add(item4);
+        items2.add(item5);
+        items2.add(item6);
+        items2.add(item7);
+        items2.add(item8);
+        items2.add(item9);
+        items2.add(item10);
+
+        inventory = new Inventory(inventoryname, items2, orders);
     }
 
+    /**
+     * Asserts that the two files of the exported inventory exists at the specificed location
+     * @throws IOException as required by Exporter
+     */
+
     @Test
     public void AssertFilesExist() throws IOException {
-        Exporter exporter = new Exporter(inventory);
+        ExportDataWrapper wrapper = new ExportDataWrapper(inventoryViewModel.getItemList(),
+                items);
+        Exporter exporter = new Exporter(inventoryname, wrapper);
         exporter.export();
-        Path p1 = Path.of("src/main/java/exports/user_inventory.txt");
+        Path p1 = Path.of("src/main/java/exports/" + inventoryname + ".txt");
```

Comment:
Is it possible to have the user specify the export path instead of hard coding it in?